src/components/atomic/button.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { styled } from "styled-components";
+
+const ButtonStyle = styled.button`
```

Comment:
Something I just noticed, but we need `:hover` and `:active` styling for this

Reply:
wasn't sure how we'd want to style that and figured i'd just leave that for when the buttons are actually made functional

Reply:
Sounds good haha